integration-tests/src/test/java/com/pip/chatbot/integration/tests/tests/weather.feature

```diff
@@ -0,0 +1,83 @@
+Feature: Restaurants Api
+
+  Background:
+    * url baseUrl
+    * def initData =
+  """
+  function() {
+    var DbUtils = Java.type('com.pip.chatbot.integration.tests.WeatherDbUtils');
+    var db = new DbUtils(dbConfig);
+    db.initWeatherData();
+    return db;
+  }
+  """
+    * def db = callonce initData
+    * configure afterFeature =
+    """
+      function(){
+      db.clearWeatherData();
+      }
+    """
+
+
+  Scenario: Integration test for insert country
+    * header Authorization =  call read('basic-auth.js') {user: #(user), password: #(password)}
+    Given path '/admin/forecasts/countries'
+    And request { "country": "CountryToUpdate" }
+    When method POST
+    Then status 200
+
+  Scenario: Integration test for update country
```

Comment:
Scenariusz myślę, że lepiej jak nazwiemy w konwencji Update country. To że to test to wiemy z kontekstu:)